### PR TITLE
cmd/geth: tidied up the source

### PR DESCRIPTION
**Pull request by kenji-isuntv**
- Created `misccmd.go`
- Moved `makedag()`, `version()`, and `license()` to `misccmd.go`
- Moved `initGenesis()` to `chaincmd.go`
- Introduced the variables `initCommand`, `makedagCommand`, `versionCommand`, and `licenseCommand` to represent the command objects
- Updated `import`s
- ~~Updated copyright year at the top~~

This is my second step to clean up the code in `cmd/geth`. The next step will be for the code logic.

### CODE REVIEW

File: cmd/geth/chaincmd.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 The go-ethereum Authors
+// Copyright 2013-2016 The go-ethereum Authors
```
Comment (karalabe) [inline cmd/geth/chaincmd.go]: Please don't update the copyright years in source files. They should be the year when the file was added.

File: cmd/geth/misccmd.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
+// Copyright 2013-2016 The go-ethereum Authors
```
Comment (karalabe) [inline cmd/geth/misccmd.go]: Please use 2016 only for files added this year.

File: cmd/geth/chaincmd.go (hunk 1 above)
Comment (kenji-isuntv) [inline cmd/geth/chaincmd.go]: Okay, got it.

File: cmd/geth/misccmd.go (hunk 2 above)
Comment (kenji-isuntv) [inline cmd/geth/misccmd.go]: Okay, got it.

**Comment by karalabe** [comment]
Sorting this PR out got a bit messy. I cherry picked your commits into a fresh PR at https://github.com/ethereum/go-ethereum/pull/3385 and will close this.

Next time please don't add Merge commits on top of an open PR. My recommendation is to use a separate feature branch for everything you'd like added, open a PR with that and leave the branch alone from that point onward so we can review and merge without issues.

**Pull request closed without merging**